src/dom_util.js

```diff
@@ -33,11 +33,10 @@ const isDocumentRoot = function(node) {
  */
 const getAncestry = function(node, root) {
   const ancestry = [];
-  let cur = node;
 
-  while (cur !== root) {
-    ancestry.push(cur);
-    cur = cur.parentNode;
+  while (node !== root) {
```

Comment:
I'm not a fan of reassigning function arguments. A minifier should be able to make this transformation. Will create a PR using http://eslint.org/docs/rules/no-param-reassign